lomap/mcs.py

```diff
@@ -98,6 +98,8 @@ def __init__(self, moli, molj, time: int = 20, verbose: str = 'info',
             of elemental changes is minimised. default False.
         element_change : bool, optional
             whether to allow elemental changes in mappings, default True
+        seed : string, optional
+            seed for MCS
```

Comment:
Might need a bit more information than that 😅

Reply:
I'm not really too keen on spending time documenting a thing that we're a) not anticipating that anyone is using directly and b) we're planning on killing off soon

Reply:
```suggestion
            Initial SMARTS seed for MCS search.
```
There fixed it for you.